run.py

```diff
@@ -138,6 +138,7 @@ def _validate_devices(devices: str):
                         help="Profiling comma separated list of activities such as cpu,cuda.")
     parser.add_argument("--cudastreams", action="store_true",
                         help="Utilization test using increasing number of cuda streams.")
+    parser.add_argument("--bs", type=int, help="Specify batch size to the test.")
```

Comment:
Add an argument to specify batch size for models that supports this argument (currently only regnet and resnest support this).